app/helpers/admin/claims_helper.rb

```diff
@@ -0,0 +1,24 @@
+module Admin
+  module ClaimsHelper
+    include ::ClaimsHelper
+
+    def eligibility_answers(eligibility)
```

Comment:
I'm uneasy with this method having exactly the same name as a method in `ClaimsHelper` as it means one will replace the other if both modules are included in the same place. Same goes for the `identity_answers` method below. This could lead to weird and wonderful bugs. Safer to give these methods a different name.

I'm also not 100% convinced of the value of having these helpers in the first place vs just having plain HTML markup with the values interpolated right there in the template. I guess having this helper does mean we can write a test that verifies the correct labels and values are being generated. The advantage of not having the method is that it gives us much more flexibility to change the view that an admin sees without being coupled to this Ruby code. When we're so early in the stages of defining the UI for admins, that flexibility might be useful as it won't preclude the UI having to display the information as a definition list. What do you think?

Reply:
I'm all for that, I only really used the helper in this instance because it mirrored the way we did it before. I'm happy to look at moving the logic to the view, especially if we back this up with view tests.

Reply:
I've had a crack at this, and I'm not sure what the benefit of adding all the logic to the view is. It just seems to complicate things more and clog up the views. I do take your point on the naming of the helpers. Better to just prefix the helpers with `admin_` rather than add the complication and possible name collision.